sdk/compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -953,14 +958,16 @@ withMaybeConfig withConfig handler = do
       ConfigFileInvalid _ (Y.InvalidYaml (Just (Y.YamlException exc))) | "Yaml file not found: " `isPrefixOf` exc ->
         pure Nothing
       ConfigFileInvalid _ (Y.InvalidYaml (Just (Y.YamlException exc))) | "packageless daml.yaml" `isInfixOf` exc -> do
-        putStrLn "Found daml.yaml with only sdk-version, ignoring this file."
+        hPutStrLn stderr "Found daml.yaml with only sdk-version, ignoring this file."
```

Comment:
We convert this to a print to stderr because when this runs inside a `daml ide` invocation printing to stdout breaks the LSP client.

While we're at it, we convert a bunch of `putStrLn` to `hPutStrLn` because we've been planning to do this for a while anyways.